vignettes/bds_finding.Rmd

```diff
@@ -342,17 +344,43 @@ dataset_vignette(
 )
 ```
 
-Similarly, for `ADEG` domain, the parameters `QTCBF` `QTCBS` and `QTCL` can be
+Similarly, for `ADEG`, the parameters `QTCBF` `QTCBS` and `QTCL` can be
 created with a function call. See example below for `PARAMCD` = `QTCF`.
 
 ```{r eval=FALSE}
-adeg <- derive_param_qtcf(
+adeg <- derive_param_qtc(
```

Comment:
We should change the heading to this section to - see above

Reply:
Nevermind, can't make suggestion in code if it hasnt been modified.   Was thinking - 
Derive Additional Parameters (BSA, BMI, MAP for ADVS, QTCF for ADEG and WBC_ABS for ADLB) - not sure if people will know WBC_ABS...

Reply:
@yangc71 What do you think about this new title?  After that I am happy to merge into devel

Reply:
Sorry I misunderstood this comment previously and thought you meant it can be ignored. The current title says **e.g.** BSA, BMI etc., so I thought it's ok to keep as it is since it's just giving examples of new parameters, so we don't have to list out everything? Also WBC_ABS is not standard term so can be confusing to include in the title.

Reply:
Sounds good!  Merging in!